internal/cluster/refresh_status_preprocessor.go

```diff
@@ -91,9 +92,19 @@ func (r *refreshPreprocessor) preprocess(ctx context.Context, c *clusterPreproce
 	for _, condition := range r.conditions {
 		stateMachineInput[condition.id.String()] = condition.fn(c)
 	}
+	for _, validationResults := range validationsOutput {
+		sortByValidationResultID(validationResults)
+	}
 	return stateMachineInput, validationsOutput, nil
 }
 
+func sortByValidationResultID(validationResults []ValidationResult) {
```

Comment:
Could you write a comment in each of these functions also mentioning the type of the values being compared? Maybe mentioning it in the name?

https://github.com/openshift/assisted-service/pull/1366#discussion_r599531894

I'm sure someone will try to unify these two functions in the future.

Reply:
Done